rest-api/src/main/java/gov/cms/qpp/conversion/api/model/UnprocessedCpcFileData.java

```diff
@@ -0,0 +1,73 @@
+package gov.cms.qpp.conversion.api.model;
+
+import java.util.Date;
+
+/**
+ * Model to hold converted metadata to Unprocessed Cpc file data.
+ */
+public class UnprocessedCpcFileData {
+	private String fileId;
+	private String filename;
+	private String apm;
+	private Date conversionDate;
+	private Boolean validationSuccess;
+
+	/**
+	 * Constructor to transform metadata into unprocessed cpc file data
+	 *
+	 * @param metadata object to be transformed
+	 */
+	public UnprocessedCpcFileData(Metadata metadata) {
+		this.fileId = metadata.getSubmissionLocator();
```

Comment:
I'm thinking that the `fileId` should be the `Uuid` of `metadata` instead of the submission locator.  This will allow quick look-up when the CPC+ team needs to mark a file as processed.  It'll be a lot harder to find (read: a scan operation) the DynamoDB item given the submission locator instead of the direct `Uuid`.

Reply:
👍

Reply:
@halprin do you think we still need the submission locator? Because the 2nd endpoint we need to get the actual file from s3.

Reply:
My thought would be that we could do a quick look-up of the `SubmissionLocator` in DynamoDB given this new `fileId` (AKA `Uuid`).

Reply:
Talked with @halprin and agreed to simply pass `Uuid` into a `fileId`